lib/chef/win32/security/sid.rb

```diff
@@ -261,23 +261,23 @@ def self.current_user
 
         SYSTEM_USER = SERVICE_ACCOUNT_USERS + BUILT_IN_GROUPS
 
-        # Сheck if the user belongs to service accounts category
+        # Check if the user belongs to service accounts category
         #
         # @return [Boolean] True or False
         #
         def self.service_account_user?(user)
           SERVICE_ACCOUNT_USERS.include?(user.to_s.upcase)
         end
 
-        # Сheck if the user is in builtin system group
+        # Check if the user is in builtin system group
         #
         # @return [Boolean] True or False
         #
         def self.group_user?(user)
           BUILT_IN_GROUPS.include?(user.to_s.upcase)
         end
 
-        # Сheck if the user belongs to system users category
+        # Check if the user belongs to system users category
```

Comment:
These are not the latin letter `C`, they are a Cyrillic character. https://en.wikipedia.org/wiki/Es_(Cyrillic)